test/fakeintake/server/server.go

```diff
@@ -83,11 +84,19 @@ func NewServer(options ...Option) *Server {
 		urlMutex:                  sync.RWMutex{},
 		clock:                     clock.New(),
 		retention:                 15 * time.Minute,
-		store:                     serverstore.NewStore(),
 		responseOverridesMutex:    sync.RWMutex{},
 		responseOverridesByMethod: newResponseOverrides(),
+		server: http.Server{
+			Addr: ":0",
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
To always prevent the macos security popup
```suggestion
			Addr: "0.0.0.0:0",
```

Reply:
Can be addressed in a follow up